march_hardware_interface/src/march_hardware_interface.cpp

```diff
@@ -86,13 +86,6 @@ void MarchHardwareInterface::init()
 
   initiateIMC();
 
-  // Print all joint positions on startup in case initialization fails.
-  this->read();
-  for (int i = 0; i < num_joints_; ++i)
-  {
-    ROS_DEBUG("[%s] First read position: %f", joint_names_[i].c_str(), joint_position_[i]);
-  }
-
```

Comment:
Kan dit niet best handig zijn? Of komt dit sowieso al in de rosbag

Reply:
This was the whole point of the PR. When these lines are executed, the iMC is not in SWITCHED_ON state and it is therefore not correct to use the read function.